test/celeritas/optical/Cerenkov.test.cc

```diff
@@ -171,18 +182,23 @@ TEST_F(CerenkovTest, angle_integral)
 
 TEST_F(CerenkovTest, dndx)
 {
-    EXPECT_SOFT_NEAR(369.81e6,
-                     constants::alpha_fine_structure * units::Mev::value()
-                         / (constants::hbar_planck * constants::c_light),
-                     1e-6);
+    if (CELERITAS_UNITS == CELERITAS_UNITS_CGS)
+    {
+        EXPECT_SOFT_NEAR(369.81e6,
+                         constants::alpha_fine_structure * units::Mev::value()
+                             / (constants::hbar_planck * constants::c_light),
+                         1e-6);
+    }
```

Comment:
Or some variation of (units are 1 / MeV-cm):
```suggestion
EXPECT_SOFT_NEAR(369.81e6,
                 constants::alpha_fine_structure * units::Mev::value()
                     / (constants::hbar_planck * constants::c_light)
                     * units::centimeter,
                 1e-6);
```